examples/blink/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Hardware
 
-Attach an LED and a current limiting resistor between GPIO 2 and 3V3 pins of an ESP32-C3 development board.
+Attach an LED and a current limiting resistor between GPIO 2 and 3V3 pins of an ESP32-C3 development board or ESP32-H2 development board.
```

Comment:
Here, again, it might make sense to change the sentence so that it doesn't turn into "of an ESP32-C3 development board or ESP32-H2 development board or ESP32-C6 development board" after the next PR.

Reply:
Yes, no problem.

Now I know that I can use `chip` as a formal name of any esp32 chip, so I will fix all README files.